src/test/java/com/hillel/items_exchange/controller/PageControllerTest.java

```diff
@@ -25,4 +25,20 @@ public void uponRequestTemp_redirectsToThePageTemp() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(view().name("temp"));
```

Comment:
You should completely remove this test as well.

Reply:
test remotely